packages/runtime/container-runtime/src/test/fuzz/summarizerFuzzMocks.ts

```diff
@@ -134,19 +134,19 @@ export class MockContainerRuntimeForSummarizer
 	/**
 	 * Prepare a SummaryNack to be sent by the server
 	 */
-	public prepareSummaryNack() {
+	public prepareSummaryNack(): void {
 		this.nackScheduled = true;
 	}
 
 	/**
 	 * Call on the Summarizer object to summarize
 	 */
-	public async summarize() {
+	public async summarize(): Promise<void> {
 		const result = this.summarizer.summarizeOnDemand({
 			reason: "fuzzTest",
 			retryOnFailure: false,
 		});
-		return Promise.all([
+		await Promise.all([
```

Comment:
I'd complain that this changes runtime behavior (the caller can no longer access each of the promise results to potentially act on them) but since we were not doing that anyway and this is test code, seems fine to make the change.

Reply:
That was my thought process exactly :)